build_tools/azure/posix-docker.yml

```diff
@@ -84,6 +85,7 @@ jobs:
         -e OPENBLAS_NUM_THREADS=$OPENBLAS_NUM_THREADS
         -e SKLEARN_SKIP_NETWORK_TESTS=$SKLEARN_SKIP_NETWORK_TESTS
         -e BLAS=$BLAS
+        -e SELECTED_TESTS=$SELECTED_TESTS
```

Comment:
> Also do you know why the linux docker job fails because I'm just trying to add a new env var ?

The [error is weird](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=40421&view=logs&jobId=aabdcdc3-bb64-5414-b357-ed024fe8659e&j=aabdcdc3-bb64-5414-b357-ed024fe8659e&t=8f942087-1e8f-5014-9da1-292eb81d9752). My guess is that the brackets (`[]`) in `$SELECTED_TESTS` is making bash parse the command weirdly. Maybe this works:

```suggestion
        -e SELECTED_TESTS="$SELECTED_TESTS"
```